common/djangoapps/adaptive_learning/config/tests/test_models.py

```diff
@@ -0,0 +1,36 @@
+"""
+Tests for configuration models of Adaptive Learning
+"""
+
+import ddt
+
+from django.test import TestCase
+from opaque_keys.edx.locator import CourseLocator
+
+from adaptive_learning.config.models import AdaptiveLearningEnabledFlag
+from adaptive_learning.config.tests.utils import adaptive_learning_enabled_feature_flags
+
+
+@ddt.ddt
+class AdaptiveLearningEnabledFlagTest(TestCase):
+    """
+    Tests the behaviour of the feature flags for Adaptive Learning
+    which are set in Django Admin
```

Comment:
@tomaszgy Missing period at the end of the sentence. Let's do "which are set via Django admin settings." here.

Reply:
:+1